cmd/mailchain/internal/http/config.go

```diff
@@ -35,6 +40,7 @@ func produceConfig(s *settings.Root) (*config, error) { //nolint: funlen
 	if err != nil {
 		return nil, errors.WithMessage(err, "Could not config sent store")
 	}
+	cacheStore := cachestore.NewCacheStore(1 * time.Hour)
```

Comment:
follow the pattern of creating a settings value that the will provide the `duration` and the `path`